enterprise_access/apps/content_assignments/admin.py

```diff
@@ -13,8 +13,8 @@ class AssignmentConfigurationAdmin(DjangoQLSearchMixin, SimpleHistoryAdmin):
     Admin configuration for AssignmentConfigurations.
     """
     list_display = (
-        'enterprise_customer_uuid',
         'uuid',
```

Comment:
Opting to treat `uuid` as the clickable cell to go into the detail. It felt a bit odd clicking on an enterprise customer UUID to go into a specific content assignment.